RecoLocalMuon/GEMRecHit/python/gemRecHits_cfi.py

```diff
@@ -14,5 +14,5 @@
 from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
 from Configuration.Eras.Modifier_phase2_GEM_cff import phase2_GEM
 
-run3_GEM.toModify(gemRecHits, ge21Off=True)
-phase2_GEM.toModify(gemRecHits, ge21Off=False)
+run3_GEM.toModify(gemRecHits, ge21Off=True, applyMasking=True)
```

Comment:
@yeckang

Afaik, the corresponding module in the Run-3 HLT menu is as follows (see, for example, [here](https://raw.githubusercontent.com/cms-sw/cmssw/7865815b9c4bb05a1b4a0adb1e1f8f89b899599e/HLTrigger/Configuration/test/OnLine_HLT_GRun.py)).
```py
process.hltGemRecHits = cms.EDProducer( "GEMRecHitProducer",
    recAlgoConfig = cms.PSet(  ),
    recAlgo = cms.string( "GEMRecHitStandardAlgo" ),
    gemDigiLabel = cms.InputTag( "hltMuonGEMDigis" ),
    applyMasking = cms.bool( False ),
    ge21Off = cms.bool( False )
)
```
Will this need to be updated at any point in 2024 (or later) ?